### PR TITLE
Edited Some Paragraphs

### PR DESCRIPTION
**Pull request by Coppertine**
Included file Template for the directory to a .osu file
Included maximum title and artist limit of 81 characters
Included Special Case of umlauts which is only on case-by-case
Included use of / \ to separate track names. (this can be removed as it can be applied in "If your track is a mashup or something among those lines"
Beatmaps must be hit sounded -> Beatmaps must be hit-sounded
Hit sounds must be audible -> Hit-sounds must be audible
Avoid using storyboarded hit sounds -> Avoid using storyboarded hit-sounds
Included example for "Also, storyboarded hit sounds at places where there are no objects can confuse the player into thinking there was some unseen hit object, which does not make sense either."
Also, storyboarded hit sounds at places -> Also, storyboarded hit-sounds at places

Add any details pertaining to wiki contributors above the break.

- [ ] Depends on #PR
- Closes #ISSUE

---

Add a sentence or two describing this change in plain english (or leave empty if the title is self-explanatory). This will be displayed on the [changelog](https://osu.ppy.sh/home/changelog). A single screenshot or short gif is also welcomed if appropriate.

### CODE REVIEW

**Comment by pishifat** [comment]
what are you doing

**Comment by Coppertine** [comment]
Forgetting about how the Ranking Criteria is edited?
